CHANGES/8365.feature

```diff
@@ -0,0 +1 @@
+Adding ansible 3 support
```

Comment:
We need to list dropping Ansible 2.8 support under removal. (Although it may technically work, but still.)

Reply:
done!